pkg/controller/pxcbackup/controller.go

```diff
@@ -232,6 +233,13 @@ func (r *ReconcilePerconaXtraDBClusterBackup) Reconcile(_ context.Context, reque
 		}
 
 		s3status = &bcpStorage.S3
+	case api.BackupStorageAzure:
+		destination = "azure://" + bcpStorage.Azure.ContainerName + "/" + cr.Spec.PXCCluster + "-" + cr.CreationTimestamp.Time.Format("2006-01-02-15:04:05") + "-full"
```

Comment:
Using `azure://` is wrong IMO, since Azure storage URIs don't work with it.

Reply:
@hors what do you think?

Reply:
yes, it the Azure storage can't work with it why we need to have it

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/1176/commits/a59e529ca5705bb2f56bc3813e7aae64b8dd28c7